src/ipa_pc/mod.rs

```diff
@@ -828,36 +1153,37 @@ impl<G: AffineCurve, D: Digest> PolynomialCommitment<G::ScalarField> for InnerPr
 
         // h(X) polynomial added to the set of polynomials for multi-poly single-point batching
         let mut labeled_polynomials = labeled_polynomials;
+        let batch_degree = batch_polynomial.degree();
         let labeled_batch_polynomial = LabeledPolynomial::new(
             format!("Batch"),
             batch_polynomial,
             None,
-            None
+            if has_hiding { Some(batch_degree) } else { None }
```

Comment:
The hiding bound for polynomials is the maximum number of query points allowed. 
Typically a polynomial is queried at a single point only. In our Boneh multi-point open, each of the polynomials in question is evaluated at least a second time, due to the new challenge point  `x`.
As I cannot think of any reason why the "batch" poly should be queried at any other point than `x` (it is specifically constructed for that single proof) let us set the hiding bound to `1`.

Reply:
Done